proto/comdex/liquidationsV2/v1beta1/liquidate.proto

```diff
@@ -0,0 +1,199 @@
+syntax = "proto3";
+package comdex.liquidationsV2.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+
+option go_package = "github.com/comdex-official/comdex/x/liquidationsV2/types";
+option (gogoproto.equal_all) = false;
+option (gogoproto.goproto_getters_all) = false;
+
+
+message LiquidationWhiteListing {
+
+  uint64 app_id = 1 [
+    (gogoproto.customname) = "AppId",
+    (gogoproto.moretags) = "yaml:\"app_id\""];
+
+  //AuctionTriggerer
+  //External Apps                  -------------> If external triggerer , then some params will be used from Comdex base app // If internal app triggerrer then params to be used from base app 
+  //Comdex Apps
+  //bool param 
+  //true - comdex apps
+  //false external apps
+  bool initiator = 6 [
+    (gogoproto.customname) = "Initiator",
+    (gogoproto.moretags) = "yaml:\"initiator\""];
+  //Sets of Params for Dutch Auction
+  bool is_dutch_activated = 7  [
+    (gogoproto.customname) = "IsDutchActivated",
+    (gogoproto.moretags) = "yaml:\"is_dutch_activated\""];
+  DutchAuctionParam dutch_auction_param = 8[
+    (gogoproto.customname) = "DutchAuctionParam",
+    (gogoproto.moretags) = "yaml:\"dutch_auction_param\""];
+  //Sets of Params for English Auction
+  bool is_english_activated = 9  [
+    (gogoproto.customname) = "IsEnglishActivated",
+    (gogoproto.moretags) = "yaml:\"is_english_activated\""];
+
+  EnglishAuctionParam english_auction_param = 10[
+    (gogoproto.customname) = "EnglishAuctionParam",
+    (gogoproto.moretags) = "yaml:\"english_auction_param\""];
+  //One thing to keep in mind that somehow it should not happen that a void is created where something at level 2 gets triggerred and it has no data saved a level 1 for lookup and it fails .
+
+
+}
+
```

Comment:
Auction params are specific to apps only or have we thought asset specific parameters also ?

Reply:
There exist two types of auction params, one is for app-level that can be controlled by them, and the other one is comdex level which protocol will control.
Bid.proto in auctionsV2 proto contains - protocol level param - AuctionParams
liquidate.proto in liquidationsV2  proto contains - app level param - LiquidationWhiteListing